Code/GraphMol/Fingerprints/MHFP.h

```diff
@@ -0,0 +1,302 @@
+//
+//  2019, Daniel Probst, Reymond Group @ University of Bern
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+/*! \file MHFP.h
+
+*/
+#include <RDGeneral/export.h>
+#ifndef __RD_MHFPFPS_H__
```

Comment:
please don't use include guards that start and end with `__`, that's for reserved symbols. (yes, there are loads of places in the RDKit where we haven't fixed this yet).
Please just use `RD_MHFPFPS_H`